packages/components/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 ### Bug Fixes
 
+-   `ComboboxControl`: Fix ComboboxControl reset button when using the keyboard. ([#63410](https://github.com/WordPress/gutenberg/pull/63410))
```

Comment:
We will need to move this CHANGELOG entry to the `Unreleased` section above, in a new `Bug fixes` sub section